doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -135,6 +135,11 @@ RF Front-End Modules
   The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
   You cannot use other MPSL features when this option is enabled.
 
+Secure Partition Manager (SPM)
+==============================
+
+* The Secure Partition Manager (SPM), kconfig option :kconfig:option:`CONFIG_SPM`, is now deprecated. It is replaced by Trusted Firmware-M (TF-M) as the supported solution.
```

Comment:
This could have a link to TF-M (user guide or kconfig or something)

Reply:
done. Cargo cult copied from elsewhere in this file as I didn't know how to make the link.